tcping.go

```diff
@@ -387,7 +433,14 @@ func tcping(tcpStats *stats) {
 		/* if the previous probe was successful
 		and the current one failed: */
 		if !tcpStats.wasDown {
+			/* Update startOfDowntime */
 			tcpStats.startOfDowntime = getSystemTime()
+
+			/* Calculate the longest uptime */
+			endOfUptime := getSystemTime()
+			calcLongestUptime(tcpStats, endOfUptime)
+			tcpStats.startOfUptime = time.Time{}
```

Comment:
What is the purpose of this line? What happens if it is not there? Since it is just returning a default time.
Same is applicable to line 468 https://github.com/pouriyajamshidi/tcping/pull/11/files#diff-e9a91df070494c5b601bd3772cf5599b25e95d52f87ec1efbbc40d8774bd6811R468